app/src/components/GrantListWithFilter.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <!-- General filters -->
+  <BaseFilterNav :items="grantRegistryListNav" :button="button" title="grants:" />
+  <GrantList :grants="sortedGrants" :grantMetadata="grantMetadata" />
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, PropType, ref, onUnmounted } from 'vue';
+// --- App Imports ---
+import BaseFilterNav from 'src/components/BaseFilterNav.vue';
+import GrantList from 'src/components/GrantList.vue';
+// --- Store ---
+import useCartStore from 'src/store/cart';
+// --- Types ---
+import { FilterNavButton, FilterNavItem, Grant, GrantMetadataResolution } from '@dgrants/types';
+import { SortingMode } from 'src/types';
+// --- Utils ---
+import { bigNumberComparator } from 'src/utils/sorting';
+
+const defaultSortingMode = 'descending';
+const grantsSortingMode = ref<SortingMode>(defaultSortingMode);
+const shuffleNonce = ref(1);
+
+const filterNavTag = computed(() => {
+  switch (grantsSortingMode.value) {
+    case 'descending': {
+      return 'newest';
```

Comment:
Why do we alias `descending` to `newest`, `random` to `shuffle`, etc? Can we just use the user-facing words as the sort mode names? Seems cleaner that way